experimental/JuLie/schur_polynomials.jl

```diff
@@ -0,0 +1,304 @@
+################################################################################
+# Schur Polynomials
+#
+# Copyright (C) 2020 Ulrich Thiel, ulthiel.com/math
+#
+# Originally taken from the JuLie [repository](https://github.com/ulthiel/JuLie)
+# by Tom Schmit and Ulrich Thiel; OSCAR-ified by Claudia He Yun and Matthias Zach.
+################################################################################
+
+export schur_polynomial
+
+@doc Markdown.doc"""
+    schur_polynomial(lambda::Partition{T}, n=sum(lambda)::Int; ring::FmpzMPolyRing) where T<:Integer
+    schur_polynomial(R::FmpzMPolyRing, lambda::Partition{T}, n=sum(lambda)::Int) where T<:Integer
+    schur_polynomial(lambda::Partition{T}, x::Vector{fmpz_mpoly}) where T<:Integer
+
+Returns the Schur polynomial ``s_λ(x_1,x_2,...,x_n)`` in n variables, as a Multivariate Polynomial.
+
+If neither `R` nor `x` are given, the Schur polynomial will be over `PolynomialRing(ZZ,["x1","x2",...,"xn"])`.
+
+# Example
+```julia-repl
+julia> R,x = PolynomialRing(ZZ, ["a","b","c"])
+(Multivariate Polynomial Ring in a, b, c over Integer Ring, fmpz_mpoly[a, b, c])
+julia> schur_polynomial(Partition([2,1]),[x[1],x[2]])
+a^2*b + a*b^2
+julia> schur_polynomial(Partition([2,1]),R)
```

Comment:
This example does not actually work (argument order is switched), which highlights the importance of using `jldoctest`